### PR TITLE
{lib,vis}[intel/2016b] POV-Ray v3.7.0.0, SDL2 v2.0.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2017.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c7acaa61add25e1046719bb3fcfbb06c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4edaf50ecc8d847967036aa251ab0a48 for a full test report.

File: easybuild/easyconfigs/s/SDL2/SDL2-2.0.4-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+description = "SDL: Simple DirectMedia Layer, a cross-platform multimedia library"
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['http://www.libsdl.org/release/']
+sources = [SOURCE_TAR_GZ]
+
+builddependencies = [('Yasm', '1.3.0')]
+
+configopts = ' --disable-nasm --without-x '
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SDL2/SDL2-2.0.4-intel-2016b.eb]: we can just as well build this with x ?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2017.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/96ad5636e0b2a1d1b4a71e011ba9b76b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0687a7c53cc59698edf3c324e0e9281f for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**